artifacts/300/deployment.json

```diff
@@ -0,0 +1,7 @@
+    "address": "0xaECDbB0a3B1C6D1Fe1755866e330D82eC81fD4FD"
+}
```

Comment:
EOF

Reply:
This is generated by the script. All files are written without an EOF.

It isn't a valid POSIX text file, but changing all of the files to have EOFs is a separate issue that I don't want to do in this PR.